packages/paste-core/components/combobox/src/types.ts

```diff
@@ -37,21 +37,27 @@ export type HighlightedIndexChanges = {
 export interface ComboboxProps extends Omit<InputProps, 'id' | 'type' | 'value'>, Pick<BoxProps, 'element'> {
   autocomplete?: boolean;
   helpText?: string | React.ReactNode;
+  labelText: string | NonNullable<React.ReactNode>;
+  optionTemplate?: OptionTemplateFn<any>;
+  groupLabelTemplate?: (groupName: string) => React.ReactNode;
+  groupItemsBy?: string;
+  variant?: InputVariants;
```

Comment:
don't we already have `variant` from InputProps being extended above?

Reply:
Yeah, you're right! I wonder why it was doing that before? 🤔

Reply:
I'm going to leave it for now, as I'm respectfully fearful of this component and how often it breaks